### ISSUE TITLE
Custom Cards

### ISSUE THREAD

**Issue by lane-eden** [dekkerglen/CubeCobra#124]
(Adding for a discoed member) As a user, I want to be able to add custom cards to my cube. Images would be ideal, but until that happens being able save custom metadata would suffice.
Add a way to add a custom card and set its name, a casting cost, a card type, rules text, and P/T.

**Comment by Awkamess** [comment]
This is my number one requested feature. Currently using a disgusting spreadsheet to organize my cube. This is the only thing I ever wanted CT to have, I think CC can make it happen :)

**Comment by pozz94** [comment]
Now that we have custom images a card named "Custom Card" with no cost, type or effect could be added to the card database as a temporary solution. It wouldn't be searchable by the new filter but I am sure it would suffice for now.

**Comment by lane-eden** [comment]
I don't think it is a good idea to go with a halfway measure in this case. If people's image links break there will be no way to know what any of the cards in their cube do. If we allow setting rules text, etc. we could at least show the card's information in place of a missing image.

**Comment by pozz94** [comment]
It wasn't meant as a real solution but as a temporary fix that could be implemented right now as we wait for a proper one.
Once we have a proper solution it would also be nice to be able to edit regular cards since some people use errata to tweak their cubes. I imagine that it would be very very similar in implementation

**Comment by lane-eden** [comment]
Yes I know it wasn't meant as the long term solution. I am arguing against doing this temporary solution and instead waiting for the correct solution in this case. If we allow people to do this half baked version then when we do get to the full solution, we will have old users with "Custom Card" cards that aren't filled out, resulting in the issue I brought up above.

**Comment by Zekromaster** [comment]
What if CubeCobra, for now, just supported adding cards from Planesculptors? They could be then autoconverted into the internal custom card format or whatever other option is adopted as the definitive one when it's implemented, with maybe a link somewhere for the original Planesculptors card.

They have a discord bot that fetches cards by set, set version and name, and it basically just does some transformation on the card name and calls their autocard api: https://github.com/mzabsky/webdrafter/blob/master/bot/index.js

It's not as polished as scryfall's API, but until something like Scryfall comes out for custom cards, that's the best option to avoid managing a database of custom cards and still support the feature.

**Comment by dekkerglen** [comment]
I'm not interested in integrating with Planesculptors. The current scryfall integration paradigm involves importing _every_ card daily, which is not realistic for this change.

This would still take a large amount of work - but won't do it in the ideal way. I'd rather spend the extra 10% of effort and create a system better catered for our use.

**Comment by Inku92** [comment]
Any update on the feature?

**Comment by dekkerglen** [comment]
this is done
